### ISSUE TITLE
cityscapes_foggy dataset

### ISSUE THREAD

**Issue by yyfyan** [Boyiliee/AOD-Net#8]
I used AOD-Net to train and test the cityscapes_foggy dataset that including 500 training set, when I train 2epoch, the loss about 0.001. The results is not good.
Can you give me some advice to train the model in the dataset. thanks.
the dataset Link: http://people.ee.ethz.ch/~csakarid/SFSU_synthetic/

**Comment by Boyiliee** [comment]
You can try Positional Normalization to see if it helps, also you could add semantic information into your model. 

Hope this helps.
